src/pe_mailer/stats_message.py

```diff
@@ -0,0 +1,109 @@
+"""This module contains the StatsMessage class."""
+
+# Standard Python Libraries
+import datetime
+
+# Third-Party Libraries
+import pystache
+
+from .message import Message
+
+
+class StatsMessage(Message):
+    """An email message containing summary statistics for a run.
+
+    Static attributes
+    -----------------
+    Subject : str
+        The mustache template to use when constructing the message
+        subject.
+
+    TextBody : str
+        The mustache template to use when constructing the plain text
+        message body.
+
+    HtmlBody : str
+        The mustache template to use when constructing the HTML
+        message body.
+
+    """
+
+    Subject = "cyhy-mailer summary from {{date}}"
+
+    TextBody = """Greetings!
+
+Here is the cyhy-mailer summary from the run ending at {{date}}:
+{{#strings}}
+* {{string}}
+{{/strings}}
+
+Please direct feedback and questions to ncats-dev@beta.dhs.gov and/or the cyhy-mailer GitHub project.
+
+Regards,
+The VM Development Team
+Cybersecurity and Infrastructure Security Agency (CISA)
+ncats-dev@beta.dhs.gov
```

Comment:
Not sure what email address you want to use here:
```suggestion
Please direct feedback and questions to ???@??? and/or the pe-mailer GitHub project.

Regards,
The P&E Development Team
Cybersecurity and Infrastructure Security Agency (CISA)
???@???
```

Reply:
Bad copy over - updated in commit: 6f8b5d8